### ISSUE TITLE
createUser with profile-informations doesn't work with firestore

### ISSUE THREAD

**Issue by michaelzoidl** [prescottprue/react-redux-firebase#475]
**Do you want to request a *feature* or report a *bug*?**  

Bug

**What is the current behavior?**

In my project i'am using
```js
firebase.createUser({
  email: 'foobar@foobar.de',
  password: 'admin'
}, {
  firstname: 'Foo',
  lastname: 'Bar'
})
```
to create an account, everything works fine, the user gets created - instant login and the flow works - except the adding of the second props (firstname, lastname) i don't know, they just get ignored.

I configured `userProfile` and `useFirestoreForProfile` in the store:
```js
const reactReduxFirebaseConfig = {
  userProfile: 'users',
  useFirestoreForProfile: true
};

const createStoreWithFirebase = composeEnhancers(
  reactReduxFirebase(firebase, reactReduxFirebaseConfig),
  reduxFirestore(firebase)
)(createStore);
```

My firestore rules looking like that in the moment (for testing purposes i enabled everything)
```js
service cloud.firestore {
  match /databases/{database}/documents {
    match /users/{userId} {
      allow read, write;
    }
  }
}
```

I also noticed that in no request - when creating a new user - the information from the second property is passed to the database:
<img width="1269" alt="screen shot 2018-05-24 at 08 11 21" src="https://user-images.githubusercontent.com/528550/40467450-4b58be3e-5f2a-11e8-9dcf-af814dbc7284.png">

Thats how a registered user looks in the `users` firebase document:
<img width="399" alt="screen shot 2018-05-24 at 08 14 29" src="https://user-images.githubusercontent.com/528550/40467530-94fee4dc-5f2a-11e8-8ce0-9621c5b66f1c.png">

**What is the expected behavior?**

When creating an user with the `createUser` function the second property should add the passed information in the `users/123123` document in firebase (firestore)

**Comment by michaelzoidl** [comment]
Short update, i'am now using just the `updateProfile` directly after the `createUser` function, seems like thats works fine :)

```js
await createUser({ email, password });
await updateProfile({ firstname, lastname });
```

But i keep this issue open, since the documented way doesn't work

**Comment by prescottprue** [comment]
@michaelzoidl You are right about how it should work. Glad to hear you have a workaround.
